tslearn/metrics.py

```diff
@@ -21,6 +22,17 @@
 GLOBAL_CONSTRAINT_CODE = {None: 0, "": 0, "itakura": 1, "sakoe_chiba": 2}
 VARIABLE_LENGTH_METRICS = ["dtw", "gak", "softdtw"]
 
+_VALID_METRICS = ['euclidean', 'braycurtis', 'canberra', 'chebyshev',
```

Comment:
Why not import those from scikit-learn? They will probably be more reactive than we can be if new metrics are added in scipy, I guess.

Reply:
Agreed, I'll even replace scipy's cdist by scikit's pairwise_distances and let it take care directly of what metrics are allowed or not.